src/sentry/nodestore/riak_light/client.py

```diff
@@ -0,0 +1,262 @@
+"""
+sentry.nodestore.riak_light.client
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+:copyright: (c) 2010-2015 by the Sentry Team, see AUTHORS for more details.
+:license: BSD, see LICENSE for more details.
+"""
+
+from __future__ import absolute_import
+
+import socket
+from random import shuffle
+from time import time
+from threading import Lock, Thread, Event
+from Queue import Queue
+
+from requests.certs import where as ca_certs
```

Comment:
This is kinda weird imo to pull this from requests, but since requests is already a dependency, requests bundles up a `pem` file for certificate validation. So to avoid bundling our own on top, might as well yank from here. Alternatively, it also works with `certifi` automagically.